<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Airline System - Premium Portal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #0f2027, stop:0.5 #203a43, stop:1 #2c5364);
    font-family: 'Segoe UI', system-ui, -apple-system, sans-serif;
}

QWidget#centralwidget {
    background-color: rgba(255, 255, 255, 0.08);
    border-radius: 20px;
    margin: 60px;
    border: 1px solid rgba(255,255,255,0.2);
}

QLabel#headerLabel {
    font-size: 42px;
    font-weight: 700;
    color: white;
    margin-top: 40px;
    margin-bottom: 10px;
    letter-spacing: 1px;
}

QLabel#subheaderLabel {
    font-size: 16px;
    font-weight: 300;
    color: rgba(255,255,255,0.7);
    margin-bottom: 40px;
}

QGroupBox {
    border: none;
    margin: 15px;
    padding-top: 15px;
    background-color: transparent;
}

QGroupBox::title {
    color: white;
    font-size: 18px;
    font-weight: 500;
    padding-left: 5px;
}

QRadioButton {
    color: white;
    font-size: 15px;
    font-weight: 400;
    margin: 12px;
    padding: 8px 8px 8px 30px;
}

QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 2px solid rgba(255,255,255,0.7);
    background: transparent;
    left: 5px;
}

QRadioButton::indicator:checked {
    background-color: #00c6ff;
    border-color: #00c6ff;
}

QLineEdit {
    background-color: rgba(255,255,255,0.1);
    border: 1px solid rgba(255,255,255,0.2);
    border-radius: 8px;
    padding: 14px;
    font-size: 15px;
    color: white;
    min-height: 45px;
    margin: 8px 0;
}

QLineEdit:focus {
    border: 1px solid #00c6ff;
    background-color: rgba(255,255,255,0.15);
}

QLineEdit::placeholder {
    color: rgba(255,255,255,0.4);
}

QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #00c6ff, stop:1 #0072ff);
    color: white;
    border: none;
    padding: 16px 32px;
    border-radius: 8px;
    font-size: 16px;
    font-weight: 500;
    min-width: 140px;
}

QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #00c6ff, stop:1 #0062ff);
}

QPushButton#ExitButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #ff416c, stop:1 #ff4b2b);
}

QPushButton#ExitButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #ff416c, stop:1 #ff3a2b);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin" stdset="0">
     <number>40</number>
    </property>
    <item>
     <widget class="QLabel" name="headerLabel">
      <property name="text">
       <string>âœˆEltoWing Reservation</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="subheaderLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#25aef3;&quot;&gt;Welcome to Omar Eltoutongy Airline Portal&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="loginGroupBox">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="RoleGB">
         <property name="title">
          <string>SELECT YOUR ROLE</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>25</number>
          </property>
          <item>
           <widget class="QRadioButton" name="AdminRButton">
            <property name="text">
             <string>Administrator</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="AgentRButton">
            <property name="text">
             <string>Booking Agent</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="PassengerRButton">
            <property name="text">
             <string>Passenger</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Elephant</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; color:#ffffff;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="UserNameEdit">
           <property name="placeholderText">
            <string>Enter your username</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Elephant</family>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-style:italic; color:#ffffff;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="PasswordEdit">
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
           <property name="placeholderText">
            <string>Enter your password</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="LoginButton">
           <property name="text">
            <string>LOGIN</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ExitButton">
           <property name="text">
            <string>EXIT</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
